website/chimera.php

```diff
@@ -0,0 +1,86 @@
+<?
+	include ("include.php");
+	
+	if (isset($_GET['type']))
+		$type = $_GET['type'];
+	else
+		$type = "saia_chimr";
+		
+	if (isset($_GET['indexnum']))
+		$indexnum = $_GET['indexnum'];
+	else
+		$indexnum = "1";
+	
+	if ($type == "trce_m0171")
+	{
+		$title = "TRACE 171 &Aring; Mosaic and NOAA Active Regions";	
+	}
+
+        else if ($type == "saia_00171")
+	  {
+	    $title = "AIA 171&Aring and NOAA Active Regions";
+	  }
+
+	else
+	{
+		$temp_index = $index_types_def[$type];
+		$title = $temp_index." and NOAA Active Regions";	

```

Comment:
NOAA AR? I thought this was to show CHs. Also, I wouldn't repeat `"and NOAA ARs"` three times, just add it to any title outside the `if` block. This will keep the code with less repetition, and only one place to change that if NOAA becomes something else in the future.